analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ClassNamedException.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+class CustomException { }                                        // Noncompliant {{Rename this class to remove "(e|E)xception" or correct its inheritance.}}
+//    ^^^^^^^^^^^^^^^
+class Customexception { }                                        // Noncompliant
+class CustomEXCEPTION { }                                        // Noncompliant
+
+class ExceptionHandler { }                                       // Compliant - "Exception" is not at end of the name of the class
+class SimpleClass { }
+class SimpleExceptionClass: Exception { }
```

Comment:
what does this case test? did you mean

```suggestion
class SimpleExceptionClass { }
```

Like this, I understand (having `Exception` in the middle). But otherwise, we already have tests with classes inheriting from Exception below.

Or you want to add something like

```
class Foo : Exception {}
```

Reply:
I've set the following test cases:
`class SimpleExceptionClass { }` -> Exception in the middle of the name

`class AlsoExtendsIt : Exception { }` -> Name doesn't contain "Exception", but derives from System.Exception